model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -185,7 +185,9 @@ object AddressResponseAddress {
     val geo: Try[AddressResponseGeo] = for {
```

Comment:
eg:
```
object AddressIndex {
  object PostcodeAddressFile {
     case class Address()
     case class Response()
  }
 object NAG {
     case class Address()
     case class Response()
  }
}
```

Instead of names like `AddressPostcodeAddressFileAddress` (or something like that)